packages/design-system/src/atoms/tooltip/index.web.tsx

```diff
@@ -67,28 +66,26 @@ export const Tooltip: React.FC<TooltipProps> = ({
       <div ref={reference} {...getReferenceProps()}>
         {children}
       </div>
-      {isOpen && (
-        <FloatingOverlay>
```

Comment:
FloatingOverlay allows you to select text inside the Tooltip. Let's make it configurable via a prop (ex. `interactive`). If the tooltip is interactive, then we'll render inside floating overlay

Reply:
I think FloatingOverlay is for preventing pointer events outside the Tooltip. Things like selecting text or clicking buttons but outside the tooltip and it also has a prop for preventing scroll. So it's more for wrapping modals, popups or alerts than tooltips.

Here is pic showing that without the FloatingOverlay we can select the text inside the tooltip and interact with it.

<img width="260" alt="Screen Shot 2022-09-20 at 6 16 29 AM" src="https://user-images.githubusercontent.com/106546965/191233919-ae63a67a-6605-487d-947f-41d29ed65e64.png">